components/Routes/Routes-List.js

```diff
@@ -0,0 +1,13 @@
+import { routes } from "../../data/lib.js";
+
+export default function RoutesList() {
+  return (
+    <ul>
+      {routes?.map((route) => (
+        <li>
```

Comment:
You will need a unique key prop here, so you might want to consider adding an ID to your routes in the data  directory :)